pkgs/development/python-modules/bravado-core/default.nix

```diff
@@ -1,47 +1,106 @@
-{ lib, buildPythonPackage, fetchFromGitHub, python-dateutil, jsonref, jsonschema,
-  pyyaml, simplejson, six, pytz, msgpack, swagger-spec-validator, rfc3987,
-  strict-rfc3339, webcolors, mypy-extensions, jsonpointer, idna, pytest, mock,
-  pytest-benchmark, isPy27, enum34 }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fqdn
+, idna
+, isoduration
+, jsonpointer
+, jsonref
+, jsonschema
+, mock
+, msgpack
+, mypy-extensions
+, pytest-benchmark
+, pytestCheckHook
+, python-dateutil
+, pythonOlder
+, pytz
+, pyyaml
+, rfc3987
+, rfc3339-validator
+, simplejson
+, six
+, strict-rfc3339
+, swagger-spec-validator
+, uri-template
+, webcolors
+}:
 
 buildPythonPackage rec {
   pname = "bravado-core";
   version = "5.17.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "Yelp";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-okQA4YJq0lyVJuDzD8mMRlOS/K3gf1qRUpw/5M0LlZE=";
+    hash = "sha256-okQA4YJq0lyVJuDzD8mMRlOS/K3gf1qRUpw/5M0LlZE=";
   };
 
-  checkInputs = [
-    mypy-extensions
-    pytest
-    mock
-    pytest-benchmark
-  ];
-
-  checkPhase = "pytest --benchmark-skip";
-
   propagatedBuildInputs = [
-    python-dateutil
     jsonref
     jsonschema
+    msgpack
+    python-dateutil
+    pytz
     pyyaml
     simplejson
     six
-    pytz
-    msgpack
     swagger-spec-validator
 
-    # the following 3 packages are included when jsonschema (3.2) is installed
+    # the following packages are included when jsonschema (3.2) is installed
     # as jsonschema[format], which reflects what happens in setup.py
```

Comment:
We should move this to optional dependencies

Reply:
Yes, we should. Probably after the update to 4.6.0.